plutus-benchmark/marlowe/test/semantics/f339f59bdf92495ed2b14e2e4d3705972b4dda59aa929cffe0f1ff5355db8d79.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 81685303867
-| mem: 231960658})
\ No newline at end of file
+({cpu: 100225050
+| mem: 327890})
```

Comment:
That's a 815x reduction in cpu and a 707x reduction in mem. Not %, _times_.

Reply:
That's what happens when you fail immediately :joy: